about.html

```diff
@@ -268,6 +230,38 @@ <h3 class="mbr-section-title mbr-fonts-style align-center mb-4 display-2">
     </div>
 </section>
 
```

Comment:
Since you're re-working this About section - should Black Alert be on here?

Reply:
Black Alert only has the Citizen role. It is not known if he is supposed to be a member of server staff or not.